Cropping faces from image/Requirements.txt

```diff
@@ -0,0 +1,3 @@
+python=3.8.2
+opencv=opencv_version =4.4.0.46
```

Comment:
wrongly written. single `=` is not correct. Also, can you run this file using the command `pip install requirements.txt`? The file name is wrong. It is requirements.txt and not Requirements.txt.